applications/GeoMechanicsApplication/custom_elements/Pw_element.h

```diff
@@ -543,8 +533,9 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) PwElement : public Element
 
     auto MakeNodalVariableGetter() const
     {
-        return
-            [this](const Variable<double>& variable) -> Vector { return GetNodalValuesOf(variable); };
+        return [this](const Variable<double>& variable) -> Vector {
```

Comment:
The function parameter name doesn't follow the Kratos Style Guide (but this was already the case):
```suggestion
        return [this](const Variable<double>& rVariable) -> Vector {
```
My suggestion would be to only change this when you need to make other changes, too.

Reply:
done.